src/actions/upsertData.ts

```diff
@@ -0,0 +1,22 @@
+import { FetchEntries, Findex, FindexKey, IndexedValue, InsertChains, Keyword, Label, Location, UpsertEntries } from "cloudproof_js";
+
+export interface IndexedEntry {
```

Comment:
if this type is used in another file, move it in `types.ts` file. if not, remove `export`